java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._1;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ */
+
+public class Task1 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        if(scanner.hasNextFloat()){
+            milesInKilometers(scanner.nextFloat());
```

Comment:
The method accepts a double argument, but you read a float value from the input. I think we need to use the same type here

Reply:
Already change if statement to read double argument.